src/commons/error.rs

```diff
@@ -208,12 +208,13 @@ pub enum Error {
     RepositoryServerAlreadyInitialised,
 
     //-----------------------------------------------------------------
-    // RFC 8181 (publishing)
+    // Publishing
     //-----------------------------------------------------------------
     Rfc8181Validation(ValidationError),
     Rfc8181Decode(String),
     Rfc8181MessageError(rfc8181::MessageError),
     Rfc8181Delta(PublicationDeltaError),
+    PublishingObjects(String),
```

Comment:
Observation: as `fn status()` will map this to HTTP 400 Bad Request - is that the correct mapping for this error?